site/docs/patterns/preferences-dialog.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: Preferences Dialog
+layout: DetailPattern
+aliases:
+  - /salt/patterns/preferences-dialog
+data:
+  resources:
+    [
+      { href: "https://www.saltdesignsystem.com/salt/components/navigation-item/examples", label: "Navigation item", internal: false },
+      { href: "https://www.figma.com/file/N6cSYycc4z6XkCjVvc1kIA/Preferences-Dialog-Pattern?type=design&node-id=157-40560&mode=design&t=eq3OapLjaquXm3UP-0", label: "Preferences Dialog stickersheet", internal: true },
+    ]
+---
+
+A preferences dialog allows you to display, group and navigate application preferences within a responsive modal dialog window. This pattern introduces guidelines to make the preferences dialog consistent and usable.
+  
+  <Diagram
+    src="/img/patterns/preferences-dialog/preferences-dialog.png"
```

Comment:
the bottom box shadow seems to be cut off in this image

Reply:
@navkaur76 do you think you can add a bit more space under the dialog so the shadow would not look cut? This goes for most of the images.

Reply:
sorted. Thanks @navkaur76